src/clients/chunker.rs

```diff
@@ -108,28 +125,32 @@ impl Client for ChunkerClient {
         "chunker"
     }
 
-    async fn health(&self) -> HealthCheckResult {
-        let mut client = self.health_client.clone();
-        let response = client
-            .check(HealthCheckRequest { service: "".into() })
-            .await;
+    async fn health(&self) -> Result<HealthCheckResult, Error> {
```

Comment:
The return type of health endpoints did not previously have any cases where errors could happen and so the return type was not wrapped in `Result<>` until now. This caused problems with code in the new grpc macro expecting a result type. It could be worked around I think, by modifying the macro, but I was also thinking for consistency all handlers should `-> Result<...>`, where for health checks the errors are still turned into appropriate health check responses. I am undecided on this detail and whether its worth looking for a change in the macro to avoid needing to make this change